azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participant/grid/ParticipantGridCellView.kt

```diff
@@ -65,6 +66,9 @@ internal class ParticipantGridCellView(
         val videoContainer: ConstraintLayout =
             findViewById(R.id.azure_communication_ui_participant_video_view_container)
 
+        val displayNameAndMicIndicatorViewContainer: LinearLayout =
```

Comment:
If you don't need anything out of View (I think you only use visibility here) it's better to just use View, one less import and the code is more robust to layout changes.

Reply:
Sure. Changed in new commit.